volttron/platform/control.py

```diff
@@ -563,6 +563,8 @@ def find_agent_data_dir(opts, agent_uuid):
         if x.endswith("agent-data"):
```

Comment:
This condition is going to match more directory names than exactly the one we create in `aip.create_agent_data_dir`. Is that desired? If so, shouldn't `aip._get_data_dir` apply the same logic?
Either way, couldn't we skip this entire block and go directly to `create_agent_data_dir`?  In aip._get_data_dir it already checks if the dir exists and only creates if not.

Reply:
done